src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json

```diff
@@ -1,11 +1,11 @@
 {
   "Color": "Red",
   "Region": "West US",
-  "feature_management": {
+  "FeatureManagement": {
```

Comment:
What is the correct name?

Reply:
As two test use the same output, these two names are both valid.

Reply:
We can use any name, as long as they are different. This looks good to me. Thanks for fixing it!